### PR TITLE
Include search total results

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

**Comment by spetrychenko** [comment]
i would like to have `spring-data-jpa` only on `db-impl` layer. on other layers, we should come up with our own pojos

**Comment by berezkin88** [comment]
working on that

File: registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/model/PageBO.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(content, totalElements);
+    }
+
+    @Override
+    public String toString() {
+        return "PageBO{" +
+            "bos=" + content +
```
Comment (spetrychenko) [inline registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/model/PageBO.java]: rename to content

File: registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/model/PagePO.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(content, totalElements);
+    }
+
+    @Override
+    public String toString() {
+        return "PagePO{" +
+            "aspspPOS=" + content +
```
Comment (spetrychenko) [inline registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/model/PagePO.java]: rename to content

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/resource/AspspResource.java [hunk 3]
```diff
@@ -42,17 +43,18 @@ public AspspResource(AspspService aspspService, AspspTOConverter converter) {
...
+            bos = aspspService.getByIban(iban, page, size);
         } else {
-            aspspBOs = aspspService.getByAspsp(buildAspspBO(name, bic, bankCode), page, size);
+            bos = aspspService.getByAspsp(buildAspspBO(name, bic, bankCode), page, size);
         }
 
         return ResponseEntity
                        .status(HttpStatus.OK)
-                       .body(converter.toAspspTOList(aspspBOs));
+                       .header("X-Total-Elements", String.valueOf(bos.getTotalElements()))
```
Comment (spetrychenko) [inline registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/resource/AspspResource.java:56]: mvc test is missing

File: registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/model/PageBO.java (hunk 1 above)
Comment (berezkin88) [reply registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/model/PageBO.java]: done

File: registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/model/PagePO.java (hunk 2 above)
Comment (berezkin88) [reply registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/model/PagePO.java]: done

File: registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/resource/AspspResource.java (hunk 3 above)
Comment (berezkin88) [reply registry-manager-rest-server/src/main/java/de/adorsys/registry/manager/resource/AspspResource.java:56]: added

**Review by spetrychenko — APPROVED** [review]

**Pull request merged**